docs/DXIL.rst

```diff
@@ -15,9 +15,9 @@ Prior to being converted into the low-level DXIL IR, a higher level IR is genera
 
 LLVM is quickly becoming a de facto standard in modern compilation technology. The LLVM framework offers several distinct features, such as a vibrant ecosystem, complete compilation framework, modular design, and reasonable documentation. We can leverage these to achieve two important objectives.
 
-First, unification of shader compilation tool chain. DXIL is a contract between IR producers, such as compilers for HLSL and other domain-specific languages, and IR consumers, such as IHV driver JIT compilers or offline XBOX shader compiler. In addition, the design provides for conversion the current HLSL IL, called DXBC IL in this document, to DXIL.
+First, unification of shader compilation tool chain. DXIL is a contract between IR producers, such as compilers for HLSL and other domain-specific languages, and IR consumers, such as IHV driver JIT compilers or offline XBOX shader compiler. In addition, the design provides for conversion of the current HLSL IL, called DXBC IL in this document, to DXIL.
```

Comment:
At this point, I think it's confusing to refer to DXBC as the "current" HLSL IL. I think it would be more appropriate to say that it allows the conversion of the _previous_ HLSL IL to DXIL.

Reply:
> At this point, I think it's confusing to refer to DXBC as the "current" HLSL IL. I think it would be more appropriate to say that it allows the conversion of the _previous_ HLSL IL to DXIL.

Would the term "legacy" be more appropriate? In other words, is "In addition, the design provides for conversion of the legacy HLSL IL - called DXBC IL in this document - to DXIL.", an accurate statement?